.github/workflows/deploy-preview.yml

```
@@ -76,44 +76,36 @@
         
-        # Patch secrets with actual values
-        kubectl patch secret barq-api-secrets -n barq-preview --type='json' -p='[
-          {"op": "replace", "path": "/stringData/ConnectionStrings__DefaultConnection", "value": "${{ secrets.SQL_SA_PASSWORD }}"},
-          {"op": "replace", "path": "/stringData/Jwt__Key", "value": "${{ secrets.JWT__KEY }}"},
-          {"op": "replace", "path": "/stringData/FileStorage__SigningKey", "value": "${{ secrets.FILESTORAGE__SIGNINGKEY }}"},
-          {"op": "replace", "path": "/stringData/Flowable__BaseUrl", "value": "${{ secrets.FLOWABLE__BASEURL }}"}
-        ]'
+        # Apply deployments and services first
+        kubectl apply -f k8s/preview/api-deployment.yaml || echo "API deployment may already exist"
+        kubectl apply -f k8s/preview/app-deployment.yaml || echo "App deployment may already exist"
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)